### DISCUSSION TITLE
Driver for Fusion RBG keyboard

### DISCUSSION THREAD

**Discussion by rcassani** [blmhemu/opengigabyte#7] [Show and tell]
Hi @blmhemu, I forgot to mention that I wrote the Python driver to control the lights in the Aorus RGB keyboard. I'm not sure if it's the same for the keyboard in the Aero. It's in [this repo](https://github.com/rcassani/keyboard-fusion-rgb). I wonder if you can try it.

**Comment by blmhemu** [comment]
Sure @rcassani It should mostly work. I will give it a try and get back to u.
